libsolidity/formal/CHC.cpp

```diff
@@ -615,6 +623,34 @@ void CHC::externalFunctionCall(FunctionCall const& _funCall)
 	m_context.addAssertion(errorFlag().currentValue() == 0);
 }
 
+void CHC::externalFunctionCallToTrustedCode(FunctionCall const& _funCall) {
+	solAssert(m_currentContract, "");
+
+	// External call is creating a new transaction.
+	auto txOrigin = state().txMember("tx.origin");
+	state().newTx();
+	// set the transaction sender as this contract
+	m_context.addAssertion(state().txMember("msg.sender") == state().thisAddress());
+	// set the origin to be the current transaction origin
+	m_context.addAssertion(state().txMember("tx.origin") == txOrigin);
+
+	auto previousError = errorFlag().currentValue();
+
+	smtutil::Expression pred = predicate(_funCall);
+
+	auto txConstraints = m_context.state().txConstraints(*functionCallToDefinition(_funCall));
```

Comment:
Here we need to check whether `functionCallToDefinition(_funCall)` is `nullptr`, similar to what the other functions do

Reply:
I am adding the check, but when could this happen? And what should happen in that case? If similar to `externalFunctionCall` then we basically ignore it and do nothing?

Reply:
It could happen when using function pointers for example

Reply:
Yea, in that case we should do nothing, and warn that these calls are not supported. I think SMTEncoder already does the warning part